authors/apps/authentication/models.py

```diff
@@ -119,7 +122,114 @@ def get_short_name(self):
         the user's real name, we return their username instead.
         """
         return self.username
-
+    
     @property
     def get_token(self):
         return jwt_encode(self.pk)
+
+    def get_reset_token(self, days=30):
+        """
+        this method ensures that the generation of tokens is kept private
+        """
+        return self._generate_jwt_token(days)
+
+    def _generate_jwt_token(self, days):
```

Comment:
There's a method in the jwt_generator.py similar to this. I think it would be better if we used that method instead of including a new method that does the same thing.

Reply:
I removed the redundant jwt_decode methods in the user model